### PR TITLE
Fixes #1897 - refactor template new-issues

### LINKED ISSUE #1897: refactor template: new-issue.html
Refactor following:
https://github.com/webcompat/webcompat.com/blob/master/webcompat/templates/new-issue.html

with following semantics:

```
{%- extends "layout.html" -%}
{% block title %}New Issue{% endblock %}
{%- block body -%}
{% include "topbar.html" %}
{% include "home-page/form.html" %}
{% endblock %}
```

### PR DESCRIPTION
**Pull request by zoepage**
r?  @miketaylr

### CODE REVIEW

File: webcompat/templates/new-issue.html [hunk 1]
```diff
@@ -2,8 +2,5 @@
 {% block title %}New Issue{% endblock %}
 {%- block body -%}
 {% include "topbar.html" %}
-<main role="main">
-  <div class="wc-Loader js-Loader"></div>
```
Comment (miketaylr) [inline webcompat/templates/new-issue.html]: https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/js/lib/bugform.js#L83

Is there an issue on file to make sure this doesn't regress when the refactor lands?

I know we have https://github.com/webcompat/webcompat.com/issues/1529, but that's very general... and I notice the refactor is either commenting out loader images, or removing them.

Comment (miketaylr) [reply webcompat/templates/new-issue.html]: We chatted about this in a 1:1, @zoepage is going to move the loader one level deeper.

Comment (zoepage) [reply webcompat/templates/new-issue.html]: Just to add some context here, the loaders have been commented out to we don't forget them and I can look for them, when working on #1529.

**Comment by miketaylr** [comment]
r_

**Pull request merged**